src/mongocxx/collection.cpp

```diff
@@ -416,7 +416,8 @@ stdx::optional<result::replace_one> collection::replace_one(view_or_value filter
     class bulk_write bulk_op(bulk_opts);
 
     model::replace_one replace_op(filter, replacement);
-    if (options.upsert()) replace_op.upsert(options.upsert().value());
+    if (options.collation()) replace_op.collation(*options.collation());
+    if (options.upsert()) replace_op.upsert(*options.upsert());
```

Comment:
Changed all one-line conditionals in this file to use braces.  See new commit and fixup commits posted.  I also filed [CXX-1095](https://jira.mongodb.org/browse/CXX-1095) to track clang-tidy integration; see the ticket description for more details.

I changed the upsert line because the use of `optional::operator*()` is much more common than `optional::value()` in this file, and in the rest of the codebase.